assets/deepmd/system.lmp

```diff
@@ -0,0 +1,202 @@
+# LAMMPS data
```

Comment:
Let's add a converter (maybe with dpData) script instead of "hardcoding" the material here